charms/argo-controller/terraform/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "argo-controller"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
+}
+
+variable "config" {
+  description = "Map of charm configuration options"
+  type        = map(string)
+  default     = {}
+}
+
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+variable "resources" {
+  description = "Map of resources revisions"
+  type        = map(number)
```

Comment:
Thanks for the comment, I 'll add a comment for this in the code but I guess we 'll have to merge as is since 0.14 isn't released yet. Do we have any clue on when is 0.14 released?

Reply:
https://discourse.charmhub.io/t/release-planning-for-the-juju-terraform-provider-v0-15-0/15540/1 

16-Sep.